ci-operator/config/openshift/builder/openshift-builder-master.yaml

```diff
@@ -14,7 +14,7 @@ images:
 - from: base
   to: docker-builder
 promotion:
-  name: "4.0"
+  name: "4.1"
```

Comment:
So is the goal to merge this commit prior to us declaring branch, or at the same time?

Might want to merge determinize commit and the ci-operator-prowgen to enforce it first, just to avoid having to rebase here.

Reply:
Yeah, we can totally do that. We were thinking to not require CI op determinize since it's not going to be necessary on all configs and only once a release, and I don't want to give more hurdles fordevs